index.html

```diff
@@ -59,6 +59,31 @@ <h1 id="title">
         This is a work in progress.
       </p>
     </section>
+    <section>
+      <h2>
+        Introduction
+      </h2>
+      <p>
+        The HTML &lt;model&gt; element allows a website to embed interactive 3D
+        models as conveniently as any other visual media. Models are served as 
+        a standalone resource. The &lt;model&gt; element also has support for 
+        interaction and animation playback while presented within the page, 
+        and supports more spatial experiences, such as the stereoscopic 
+        display of 3D content where available.
+      </p>
+      <p>
+        As &lt;model&gt; is rendered directly by the user agent, it has
+        the ability to utilize privileged information that is not made
+        available to a Javascript context, such as the user's head and eye
+        position or the lighting of the user's environment. Additionally, the
+        use of accessibility information and controls may be engaged in without
+        divulging that information to a Javascript context.
```

Comment:
We should probably frame this as: 

```suggestion
        the ability to utilize privileged information, such as the user's head and eye
        position or the lighting of the user's environment, without exposing that information to script. 
        Additionally, the use of accessibility information and controls can be engaged in a privacy-preserving
        manner.
```